src/pages/documentation/turn/connecting.astro

```diff
@@ -32,51 +34,39 @@ const page: IStandardPageMetadata = {
   <QuickLinks links={page.section} slot="quick-links" />
   <Section {...page.section[0]}>
     <p>
-      To create a new subscription or renew an existing subscription for Nevada GPS, you'll need to provide the
-      following information:
+      Connecting to our GPS networks requires a <ExternalLink href="https://id.utah.gov/">Utah ID</ExternalLink>, and
```

Comment:
**question**: does this need to link to the new login page for public folks? @brigsz or @nathankota did you ever sort that out?

Reply:
I would guess it should go to the new login page.  If not we can go backwards, but pretty sure new login is the one

Reply:
Steve and I will confirm what we are hearing on this issue with DTS folks

Reply:
we can't use the new one yet so we can wait until they are ready to onboard us.